android/src/main/java/org/reactnative/camera/tasks/ResolveTakenPictureAsyncTask.java

```diff
@@ -239,20 +253,34 @@ private Bitmap flipHorizontally(Bitmap source) {
         return Bitmap.createBitmap(source, 0, 0, source.getWidth(), source.getHeight(), matrix, true);
     }
 
+    private int getCameraOrientation() {
+        int orientation = 0;
+        if (Build.VERSION.SDK_INT >= 21) {
+            CameraManager manager = (CameraManager) mThemedReactContext.getSystemService(Context.CAMERA_SERVICE);
+            try {
+                String cameraId = manager.getCameraIdList()[0];
```

Comment:
Are you sure this won't cause issues? Always getting the first available camera? What if the picture was taking with the front camera?

Reply:
That is nice catch. I will update the code.